docker/README.md

```diff
@@ -172,3 +171,32 @@ A convenient way to generate a desired number of random bytes and base64-encode
 ```
 $ openssl rand -base64 <desired length>
 ```
+
+## Metrics Exporter
+
+See [monitoring/exporter/README](../monitoring/exporter/README.md) for information on the Exporter.
+Container is also available as a part of the Tinode docker distribution: `tinode/exporter`.
+Run it with
+
+```
+$ docker run -p 6222:6222 -d --name tinode-exporter --network tinode-net \
+		--env SERVE_FOR=<prometheus|influxdb> \
+		--env TINODE_ADDR=<tinode metrics endpoint> \
+		--env LISTEN_AT=":6222" \
```

Comment:
Remove? Users can't really change it because of `EXPOSE 6222`. See Dockerfile comment below.

Reply:
Done.